tests/testthat/test-as_html.R

```diff
@@ -36,6 +36,11 @@ test_that("code inside Sexpr is evaluated", {
   expect_equal(rd2html("\\Sexpr{1 + 2}"), "3")
 })
 
+test_that("can convert cross links to online documentation url", {
+  expect_equal(rd2html("\\link[base]{library}"),
+               link_remote(label="library", topic="library", package="base"))
```

Comment:
Honestly, it will require some re-engineering to fix parsing of things like `\link[=terms.object]{terms}`. I'm going to make a separate issue and I will explain why there.